### ISSUE TITLE
Clicking `Add new month` doesn't work

### ISSUE THREAD

**Issue by geoidesic** [DasSauerkraut/calendar-weather#162]
I click on `Add new month` and nothing happens. Same with `Add Weekday`. The trash can button also doesn't work.

Foundry Version: 0.7.5

Console errors:
```
calendarForm.mjs:42 year46164
VM1888:1 Uncaught SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at HTMLButtonElement.<anonymous> (calendarForm.mjs:187)
    at HTMLButtonElement.dispatch (jquery.min.js:2)
    at HTMLButtonElement.v.handle (jquery.min.js:2)
(anonymous) @ calendarForm.mjs:187
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
calendarForm.mjs:138 46164
calendarForm.mjs:167 calendar-weather | Building new calendar with the following object: {months: Array(1), daysOfTheWeek: Array(4), year: 46164, day: 0, numDayOfTheWeek: 3, …}
```

**Comment by geoidesic** [comment]
![Screenshot 2020-10-27 at 23 02 08](https://user-images.githubusercontent.com/4637995/97371150-8af7cc80-18a8-11eb-81fe-eaf44c5ac423.png)

**Comment by frondeus** [comment]
Same as https://github.com/DasSauerkraut/calendar-weather/issues/154

**Comment by geilt** [comment]
Also not working. I also cannot resize the window or add any new months or really anything to the calendar. Something is up.

**Comment by geoidesic** [comment]
Fixed in this PR #163
